frontend/src/components/annotation/text_annotation/text_toolbar.vue

```diff
@@ -374,10 +374,12 @@ import label_schema_selector from "../../label/label_schema_selector.vue"
 import task_status from "../image_and_video_annotation/task_status.vue"
 import CustomButtonsSection from "@/components/ui_schema/custom_buttons_section.vue";
 import image_and_video_toolbar from "@/components/annotation/image_and_video_annotation/toolbar.vue";
+import logo from "../../diffgram/logo.vue";
 
 export default Vue.extend({
   name: "text_toolbar",
   components: {
+    logo,
```

Comment:
Component didn't have logo element registered with it, so it was throwing a runtime exception